tasm-staff-editor/src/components/ReadingLinks/ReadingLinks.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+
+export default function ReadingLinks() {
+    const [articleLink, setArticleLink] = useState('');
+
+    const handleChange = (e) => {
+        setArticleLink(e.target.value);
+    };
+
+    return (
+        <div>
+            <p className='text-center text-xl block mb-4'>Additional reading</p>
+            <label className='block mb-4'>
+                Link:
+                <input
+                    type="text"
+                    name="articleLink"
+                    value={articleLink}
+                    onChange={handleChange}
+                    className="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-lightBlue-600 sm:leading-6"
+                />
+            </label>
+            <label className='block mb-4'>
+                Link:
+                <input
+                    type="text"
+                    name="articleLink"
+                    value={articleLink}
+                    onChange={handleChange}
+                    className="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-lightBlue-600 sm:leading-6"
```

Comment:
The two `className="..."` lines here (lines 30, and 20) look alike, could we define this in our `index.css` so that it's contained in one consistent place?
Preventing any issues with it being accidentally modified down the road?
See a few places like that in `MediaType.js` as well

Reply:
I started to do that but I was more so focused on getting the class components changed to function components since the styling will still needed to updated beyond this change. That will need to be added but I would like to discuss that with Foster on Monday since I am not sure if he wants to change the styling to be based on flexbox instead of block.

Reply:
Perfect! I didn't know it that would be in scope for this PR, so knocking it out Monday sounds perfect!